adapters/contracts/loopring/loopring.go

```diff
@@ -0,0 +1,17 @@
+// Code generated - DO NOT EDIT.
+// This file is a generated binding and any manual changes will be lost.
+
+package loopring
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+)
+
+
+// LoopringMetaData contains all meta data concerning the Drips contract.
```

Comment:
this should be generated using the ./abigen.sh, add looping folder into the script and generate this file

Reply:
fixed